ec2/ec2test/server.go

```diff
@@ -1955,13 +2037,26 @@ func (srv *Server) describeIFaces(w http.ResponseWriter, req *http.Request, reqI
 	resp.XMLName = xml.Name{defaultXMLName, "DescribeNetworkInterfacesResponse"}
 	resp.RequestId = reqId
 	for _, i := range srv.ifaces {
-		ok, err := f.ok(i)
-		_, known := idMap[i.Id]
-		if ok && (len(idMap) == 0 || known) {
+		// Without neither filers nor ids - return everything.
```

Comment:
D'oh! Nice catch, but I've simplified the code as you suggested below and this is now gone.